src/aics-image-viewer/components/App/index.tsx

```diff
@@ -220,6 +220,7 @@ const App: React.FC<AppProps> = (props) => {
   const [playingAxis, setPlayingAxis] = useState<AxisName | "t" | null>(null);
   playControls.onPlayingAxisChanged = (axis) => {
     loader.current?.setPrefetchPriority(axis ? [axisToLoaderPriority[axis]] : []);
+    loader.current?.syncMultichannelLoading(axis ? true : false);
```

Comment:
this line ensures that multichannel playback waits for all channels to arrive before redrawing.

Reply:
also see #195